app/lib/case_details_section.rb

```diff
@@ -20,12 +20,12 @@ def tribunal(estimate)
       :tribunal if estimate.level_of_help == "controlled" && FeatureFlags.enabled?(:asylum_and_immigration)
     end
 
-    def matter_type(estimate)
-      :matter_type if estimate.upper_tribunal
+    def asylum_support(estimate)
+      :asylum_support if estimate.level_of_help == "controlled" && FeatureFlags.enabled?(:asylum_and_immigration) && estimate.upper_tribunal
```

Comment:
You don't need the FeatureFlags check here, because `estimate.upper_tribunal` can only be true if the flag is enabled anyway.